packages/grafana-data/src/types/datasource.ts

```diff
@@ -250,14 +249,6 @@ abstract class DataSourceApi<
    */
   getQueryDisplayText?(query: TQuery): string;
 
-  /**
-   * Retrieve context for a given log row
-   */
-  getLogRowContext?: <TContextQueryOptions extends {}>(
-    row: LogRowModel,
-    options?: TContextQueryOptions
-  ) => Promise<DataQueryResponse>;
-
```

Comment:
This is part of the public api, not sure we can just remove it like this. I think we need to deprecate it first.

Reply:
We will keep them until 9.0 🚀  #39823